### PR TITLE
Editorial: Fix some mathematical type issues around PlainDateTime.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: polyfill/lib/plaindatetime.mjs [hunk 1]
```diff
@@ -32,15 +32,15 @@ export class PlainDateTime {
...
-    isoYear = ES.ToInteger(isoYear);
-    isoMonth = ES.ToInteger(isoMonth);
-    isoDay = ES.ToInteger(isoDay);
-    hour = ES.ToInteger(hour);
-    minute = ES.ToInteger(minute);
-    second = ES.ToInteger(second);
-    millisecond = ES.ToInteger(millisecond);
-    microsecond = ES.ToInteger(microsecond);
-    nanosecond = ES.ToInteger(nanosecond);
+    isoYear = ES.ToFiniteInteger(isoYear);
```
Comment (ljharb) [inline polyfill/lib/plaindatetime.mjs:35]: Is “ToFiniteInteger” a thing in the spec text?

Comment (ptomato) [reply polyfill/lib/plaindatetime.mjs:35]: It is currently not, my guess is that @Ms2ger is planning to add it as an editorial change as part of #1413 if it makes things more readable once the other spec text changes are complete?

**Review by ptomato — APPROVED** [review]
:+1:

File: polyfill/lib/plaindatetime.mjs (hunk 1 above)
Comment (ljharb) [reply polyfill/lib/plaindatetime.mjs:35]: No concern about adding the AO, but it seems a bit strange to use it in the polyfill before there's spec text for it.

Comment (ptomato) [reply polyfill/lib/plaindatetime.mjs:35]: Since it was pre-existing, I guess this would be appropriate for a follow-up.

**Pull request merged**